ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/committer/CommitContextSpec.scala

```diff
@@ -28,14 +28,7 @@ class CommitContextSpec extends WordSpec with Matchers {
       context.get(aKey) shouldBe Some(aValue)
     }
 
-    "record accessed input keys" in {
-      val context = newInstance(Map(aKey -> Some(aValue)))
-      context.get(aKey)
-      context.getAccessedInputKeys.size shouldBe 1
-      context.getAccessedInputKeys.head shouldBe aKey
-    }
-
-    "maintain the order of accessed input keys based on when they were accessed first" in {
+    "records all accessed input keys and iterates over them in a stable order" in {
```

Comment:
@miklos-da This really checks all the properties we're interested in and it includes the previous one, so I removed the latter.

Reply:
That's fine. In order to document by a test that the input set is not necessarily the same as the read keys I would suggest adding a test case where you have 2 input keys but only read one (even if they're `None`, doesn't matter).

Reply:
Good idea, added.